turbinia/evidence.py

```diff
@@ -356,9 +372,16 @@ def __init__(self, project=None, zone=None, disk_name=None, *args, **kwargs):
     self.cloud_only = True
 
   def _preprocess(self):
-    self.device_path = google_cloud.PreprocessAttachDisk(self.disk_name)
-    self.mount_path = mount_local.PreprocessMountDisk(
-        self.device_path, self.mount_partition)
+    self.device_path, partition_paths = google_cloud.PreprocessAttachDisk(
+        self.disk_name)
+    if self.mount_partition > len(partition_paths):
```

Comment:
I wonder if we should we move this logic into PreprocessMountDisk() to reduce duplication?

Reply:
Actually. Yes.